### PR TITLE
adding tls client certificate support

### PR DESCRIPTION
**Pull request by husobee**
Fixes https://github.com/davecheney/httpstat/issues/94

This pull request adds support for client certificates.  To use specify a client certificate in PEM format that includes both the certificate and the private key in the same file with the "-c" flag.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -63,6 +64,8 @@ var (
 	redirectsFollowed int
 
 	version = "devel" // for -v flag, updated during the release process with -ldflags=-X=main.version=...
+
+	clientCertFile string
```
Comment (theckman) [inline main.go]: Can this be moved up to be with the rest of the command line flags?

File: main.go [hunk 2]
```diff
@@ -77,6 +80,7 @@ func init() {
 	flag.BoolVar(&saveOutput, "O", false, "Save body as remote filename")
 	flag.StringVar(&outputFile, "o", "", "output file for body")
 	flag.BoolVar(&showVersion, "v", false, "print version number")
+	flag.StringVar(&clientCertFile, "c", "", "client cert file for tls config")
```
Comment (theckman) [inline main.go]: What do you think about using `E` which is the same flag `cURL` uses? This project is a Go version of `httpstat.py`, which is just a wrapper around `cURL`. So we should try to keep as much parity as possible.

File: main.go (hunk 1 above)
Comment (husobee) [inline main.go]: Agree, will change.

File: main.go (hunk 2 above)
Comment (husobee) [inline main.go]: Agree, will change.

**Pull request merged**